README.md

```diff
@@ -63,17 +63,5 @@ terms specified in the [LICENSE] file.
 
 [LICENSE]: /LICENSE.md
 
-## About
-
-![thoughtbot](https://thoughtbot.com/brand_assets/93:44.svg)
-
-thoughtbot stylelint Config is maintained and funded by
-[thoughtbot, inc.][thoughtbot] The names and logos for thoughtbot are
-trademarks of thoughtbot, inc.
-
-We love open source software! See [our other projects][community] or
-[hire us][hire] to help build your product.
-
-[thoughtbot]: https://thoughtbot.com/?utm_source=github
-[community]: https://thoughtbot.com/community?utm_source=github
-[hire]: https://thoughtbot.com/hire-us?utm_source=github
+<!-- START /templates/footer.md -->
```

Comment:
I appreciate you've probably done this elsewhere by now, but the leading `/` implies to me the start of a filesystem, which it presumably isn't. Can that go?

Reply:
Hi Nick. If I remove that, the workflow can't find the file in the templates repo. I tried that before and it only worked like this. I was trying a bunch of things at the time, though. Maybe it's not needed anymore? I can try removing it and see if it works later this week.

Reply:
Oh interesting, that's weird! Let me know how you get on, I'm curious!

Reply:
Okay, thanks for your patience, Nick! I tested without the leading `/` on this test repo and it worked:

https://github.com/thoughtbot/testing-reusable-workflow

so we're good to go without it. Thanks for bringing that up. I also updated the original workflow to open a PR with the README changes because some repos have a protected branch rule.

Reply:
Removing it in https://github.com/thoughtbot/templates/pull/31

Reply:
So after merging that, the dynamic readme doesn't work. I don't really know why but removing the "/" doesn't work, so we need it.